server/router.js

```diff
@@ -24,4 +25,11 @@ module.exports = function(app) {
   });
   app.post("/signin", requireSignin, Authentication.signin);
   app.post("/signup", Authentication.signup);
+
+  app.post("/tasks/add", tasks.addtask);
+  app.get("/tasks/", tasks.getTasks);
+
+  //routing will probably change too for delete and update
+  app.delete("/tasks/:id", tasks.deleteTask);
+  app.post("/tasks/:id", tasks.updateTasks);
```

Comment:
i thought that it would be okay because there is a different request. I'm not too sure about how it works, but the tutorial I watched showed this being fine?

Reply:
There's functionally nothing wrong, since it is using `post` or `delete`. I just think that it would make it easier to understand at a glance, to avoid any complications when we call these endpoint in the future. If you think it's fine like this, then all good. :)